app/game/[gameId]/view/GameView.tsx

```diff
@@ -0,0 +1,81 @@
+"use client"
+
+import React from "react"
+import { Game } from "../../index"
+import { RemakeGame } from "./RemakeGame"
+
+interface Props {
+  game: Game
+  gameCode: string
+}
+
+export const GameView = ({ game, gameCode }: Props) => {
+  const [username, setUsername] = React.useState<string>();
+  const [shouldShowDoNotOpen, setShouldShowDoNotOpen] = React.useState<boolean>(false);
+
+  React.useEffect(() => {
+    const name = window.localStorage.getItem("username");
+    if (name) {
+      setUsername(name);
+    }
+  }, []);
+
+  return (
+    <>
+      {game &&
+        <div className="max-w-5xl w-full font-mono text-sm flex flex-col">
+          <div className="max-w-lg w-full h-full flex flex-col self-center items-center justify-center mb-6 text-lg">
+            <h2>Game Code: {gameCode}</h2>
+          </div>
+          <div className="max-w-lg w-full h-full flex flex-col self-center items-center justify-center">
+            <div className="flex flex-row justify-center items-center w-full mb-6">
+              <p className='text-md font-bold'>{game.start}</p>
+            </div>
+            <div className="flex flex-col justify-center items-center w-full mb-6">
+              {game.players.map((player, index) => {
+                if (player !== username) {
+                  return null
+                }
+                return (
+                  <div key={index} className="flex flex-row justify-center items-center w-full mb-4">
+                    <div className="max-w-3xl w-full font-mono text-sm flex flex-col">
+                      <pre className="whitespace-pre-line">
+                        {game[username]}
```

Comment:
It's kinda weird to me that this mapping of player to info isn't some field inside of game, but not a new change in this pr. It's just that the game might end up getting more complicated down the road.